src/Microsoft.OData.Core/UriParser/Binders/FunctionCallBinder.cs

```diff
@@ -590,7 +590,7 @@ private static ICollection<FunctionParameterToken> HandleComplexOrCollectionPara
                 string valueStr = null;
                 if (valueToken != null && (valueStr = valueToken.Value as string) != null && !string.IsNullOrEmpty(valueToken.OriginalText))
                 {
-                    ExpressionLexer lexer = new ExpressionLexer(valueToken.OriginalText, true /*moveToFirstToken*/, false /*useSemicolonDelimiter*/, true /*parsingFunctionParameters*/);
+                    ExpressionLexer lexer = new ExpressionLexer(model, valueToken.OriginalText, true /*moveToFirstToken*/, false /*useSemicolonDelimiter*/, true /*parsingFunctionParameters*/);

```

Comment:
Could be out of this PR scope but:
```suggestion
                    ExpressionLexer lexer = new ExpressionLexer(model, valueToken.OriginalText, moveToFirstToken: true, useSemicolonDelimiter: false, parsingFunctionParameters: true);

```